website/docs/r/virtual_machine.html.markdown

```diff
@@ -473,6 +473,7 @@ output "principal_id" {
 
 * `provision_vm_agent` - (Optional) This value defaults to false.
 * `enable_automatic_upgrades` - (Optional) This value defaults to false.
+* `timezone` - (Optional) Specifies the time zone of the virtual machine. e.g. "Pacific Standard Time".
```

Comment:
We can suffix this with ". Defaults to `UTC`"

Reply:
Done.